platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -605,32 +632,34 @@ check_package_manager() {
     not_found=0
     package_managers="apt-get dnf yum dpkg rpm"
     for package in $package_managers; do
-    {
-        res="$(need_cmd $package)"  
-        if [ $? -eq 0 ] ; then
-            not_found=0
-            wrap_debug "Current target platform supports $package package manager"
-            wrap_pass "check_package_manager"
-            if [ $package = "rpm" ] || [ $package = "dpkg" ]; then
-                check_ca_cert
+        {
+            # TODO : Is there a better way to do this?
+            res="$(need_cmd $package)"
+            if [ $? -eq 0 ]; then
+                not_found=0
+                wrap_debug_message "Current target platform supports $package package manager"
+                wrap_pass "check_package_manager"
+                if [ $package = "rpm" ] || [ $package = "dpkg" ]; then
+                    check_ca_cert
+                fi
+                break
+            else
+                not_found=1
+                wrap_debug_message "Current target platform does not support $package package manager"
             fi
-            break;
-        else
-            not_found=1
-            wrap_debug "Current target platform does not support $package package manager"
-        fi
-    }
+        }
     done
     if [ "$not_found" -eq 1 ]; then
-        wrap_warn "check_package_manager"
-        wrap_warning "IoT Edge supports the following package types [*deb, *rpm] and following package managers [apt-get].We have identified that this device does not have support for the supported package type. Please head to aka.ms/iotedge for instructions on how to build the iotedge binaries from source"
+        wrap_warning "check_package_manager"
+        wrap_warning_message "IoT Edge supports the following package types [*deb, *rpm] and following package managers [apt-get].We have identified that this device does not have support for the supported package type. Please head to aka.ms/iotedge for instructions on how to build the iotedge binaries from source"
         check_ca_cert
     fi
 }
 
 check_ca_cert() {
     if [ ! -d "/etc/ca-certificates" ]; then
-        wrap_warn "Install ca-certificates package"
+        wrap_warning "check_ca_cert"
```

Comment:
Should we also check the read/execute permissions of the directory & the cert given the current user?

Reply:
The assumption is that the user should run this script with correct priveleges,  We have a warning at the beginning pointing if user is running as root.